### PR TITLE
add overridable cmp for standard number

### PR DESCRIPTION
**Pull request by KonstantinaStoikou**
- add overridable components for standard number in frontsite card and list results view
- fix iso loan dates in search results

### CODE REVIEW

File: src/lib/modules/Document/DocumentListEntry/DocumentListEntry.js [hunk 1]
```diff
@@ -109,6 +110,10 @@ export default class DocumentListEntry extends Component {
             <LiteratureTitle title={this.metadata.title} />
           </Item.Header>
           <Item.Meta>
+            <Overridable
```
Comment (ntarocco) [inline src/lib/modules/Document/DocumentListEntry/DocumentListEntry.js:113]: this is named after authors, but it is before, no? Is it the right place?

File: src/lib/pages/frontsite/Documents/DocumentDetails/DocumentCirculation/DocumentEItems.js [hunk 2]
```diff
@@ -31,8 +31,8 @@ export class DocumentEItems extends Component {
           <DocumentEItemUrls eitems={eitems} />
         ) : (
           <>
-            No e-resources currently available.{' '}
-            <Link to={this.onClickEItemRequestLink()}>Request the e-book</Link>
+            No e-resources currently available. <br />
+            <Link to={this.onClickEItemRequestLink()}>Request the e-book.</Link>
```
Comment (ntarocco) [inline src/lib/pages/frontsite/Documents/DocumentDetails/DocumentCirculation/DocumentEItems.js:35]: 👍🏼

File: src/lib/modules/Loan/backoffice/LoanList/LoanDates.js [hunk 3]
```diff
@@ -2,23 +2,28 @@ import React, { Component } from 'react';
...
     const isRequest = invenioConfig.CIRCULATION.loanRequestStates.includes(
       loan.metadata.state
     );
+
     return isRequest ? (
       <>
         <List.Content floated="right">
-          {loan.metadata.request_start_date}
+          {loan.metadata.request_start_date &&
+            toShortDate(DateTime.fromISO(loan.metadata.request_start_date))}
```
Comment (ntarocco) [inline src/lib/modules/Loan/backoffice/LoanList/LoanDates.js:19]: 👍🏼

File: src/lib/pages/frontsite/Literature/LiteratureSearch/LiteratureSearch.js [hunk 4]
```diff
@@ -53,72 +54,82 @@ class LiteratureSearch extends Component {
...
-        }}
-      >
-        <ReactSearchKit
-          searchApi={this.searchApi}
-          history={history}
-          urlHandlerApi={urlHandler}
-          overridableId="LiteratureSearchOverridable"
-          initialQueryState={initialState}
-          defaultSortingOnEmptyQueryString={defaultSortingOnEmptyQueryString}
+      <Overridable id="LiteratureSearch.extraMap">
```
Comment (ntarocco) [inline src/lib/pages/frontsite/Literature/LiteratureSearch/LiteratureSearch.js:61]: why `extraMap`?
here we make overridable the entire search app, is this what we want?
Why not using the `LiteratureSearchOverridesMap` prop?

File: src/lib/modules/Document/DocumentListEntry/DocumentListEntry.js (hunk 1 above)
Comment (KonstantinaStoikou) [reply src/lib/modules/Document/DocumentListEntry/DocumentListEntry.js:113]: I renamed it

**Review by ntarocco — APPROVED** [review]

**Pull request merged**